.github/actions/environment/action.yml

```diff
@@ -47,9 +47,14 @@ runs:
                 free
             fi
         fi
-        # Drop a tag that points to a current commit (if any)
-        # on a job triggered by pushing to a branch.
-        if ${{ ! startsWith(github.ref, 'refs/tags/') }}; then
+        if ${{ startsWith(github.ref, 'refs/tags/') }}; then
+            # To trigger packages creation on tagged commits need to set
+            # CI_COMMIT_TAG environment variable for .gitlab.mk make file:
+            tag_name=${{ github.ref }}
+            echo CI_COMMIT_TAG=${tag_name#refs/tags/} | tee -a $GITHUB_ENV
```

Comment:
We can parse [GITHUB_REF](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) in `.gitlab.mk` (if it starts with `refs/tags/`, then we're in a tag job). This way we'll eliminate non obvious dependency between action.yml and .gitlab.mk.

Reply:
Sure, moved the check into .gitlab-ci.mk file.